tests/tools/onert_train/src/nnfw_util.h

```diff
@@ -31,11 +32,17 @@
     }                                \
   } while (0)
 
+template <typename E> constexpr auto to_underlying(E e) noexcept
+{
+  return static_cast<std::underlying_type_t<E>>(e);
+}
+
 namespace onert_train
 {
 uint64_t num_elems(const nnfw_tensorinfo *ti);
 uint64_t bufsize_for(const nnfw_tensorinfo *ti);
 
+// TODO: Rename it to use snake case (to_string)
```

Comment:
Is it safe to use the same name as [std::to_string](https://en.cppreference.com/w/cpp/string/basic_string/to_string)?

Reply:
It may be safe if we does not use `using namespace std`.